packages/mdc-select/README.md

```diff
@@ -331,9 +346,16 @@ within `componentDidUpdate`.
 | --- | --- |
 | `addClass(className: string) => void` | Adds a class to the root element. |
 | `removeClass(className: string) => void` | Removes a class from the root element. |
+| `addClassToLabel(className: string) => void` | Adds a class to the label |
+| `removeClassFromLabel(className: string) => void` | Removes a class from the label |
+| `addClassToBottomLine(className: string) => void` | Adds a class to the bottom line |
+| `removeClassFromBottomLine(className: string) => void` | Removes a class from the bottom line |
+| `setBottomLineAttr(attr: string, value: string) => void` | Adds an attribute to the bottom line |
 | `setAttr(attr: string, value: string) => void` | Sets attribute `attr` to value `value` on the root element. |
 | `rmAttr(attr: string) => void` | Removes attribute `attr` from the root element. |
 | `computeBoundingRect() => {left: number, top: number}` | Returns an object with a shape similar to a `ClientRect` object, with a `left` and `top` property specifying the element's position on the page relative to the viewport. The easiest way to achieve this is by calling `getBoundingClientRect()` on the root element. |
+| `registerPointerDownHandler(evtType: string, handler: EventListener) => void`| Adds an event listener for pointer down interactions to the root element |
```

Comment:
Maybe I lack historical context, but what's the reason for increasing the adapter API surface for this? It looks like we could just reuse `registerInteractionHandler` to register `touchstart` and `pointerdown` handlers and likewise for deregistering.